lib/python/qtvcp/widgets/led_widget.py

```diff
@@ -37,6 +37,7 @@ def __init__(self, parent=None):
         self._diameter = 15
         self._color = QColor("red")
         self._off_color = QColor("black")
+        self._border_color = QColor("red")
```

Comment:
Does border_cokor need the color() around it

Reply:
Possibly not, I just assumed that it would be needed.

Reply:
Without it this error occurs:
  File "/home/phill/linuxcnc-dev/lib/python/qtvcp/widgets/led_widget.py", line 111, in paintEvent
    painter.setPen(QColor(self._border_color))
AttributeError: 'LED' object has no attribute '_border_color'